<launch>
    
    <include file="$(find turtlebot3_gazebo)/launch/multi_turtlebot3_detectron2.launch">        
        <arg name="ns" value="tb3_0"/>
    </include> 
    <include file="$(find turtlebot3_gazebo)/launch/multi_turtlebot3_detectron2.launch">        
        <arg name="ns" value="tb3_1"/>
    </include> 
    <include file="$(find turtlebot3_gazebo)/launch/multi_turtlebot3_detectron2.launch">        
        <arg name="ns" value="tb3_2"/>
    </include> 

</launch>